components/atoms/Container.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+import { maxWidth } from 'styled-system'
+import { Box } from '@rebass/grid'
+
+// Based on suggestion by `rebass` developer
+// https://github.com/rebassjs/grid/issues/166#issuecomment-450678818
+
+const Container = styled(Box)(maxWidth)
+
+Container.defaultProps = {
+  px: 3,
+  mx: 'auto',
+  maxWidth: [768, 1024, 1280]
+}
+
+export default Container
```

Comment:
This is cool. I like!

Reply:
I put very little thought into what the values should be. They _sound_ reasonable, we need to make sure they _look_ appropriate too.